index.js

```diff
@@ -1,61 +1,69 @@
 const url = 'https://api.themoviedb.org/3/movie/12?api_key=7c4a986b16b3b892bd7111a358d63e05&language=en-US';
 const logo = 'https://api.themoviedb.org/3/movie/75780?api_key=7c4a986b16b3b892bd7111a358d63e05&language=en-US'
 const path = 'https://image.tmdb.org/t/p/w1920_and_h600_multi_faces'
+const pathCard = 'https://image.tmdb.org/t/p/w185'
 const latestRelease = 'https://api.themoviedb.org/3/trending/all/day?api_key=7c4a986b16b3b892bd7111a358d63e05'
 const pathMovie = 'https://api.themoviedb.org/3/search/movie?api_key=7c4a986b16b3b892bd7111a358d63e05&query='
 const inputBar = document.getElementById('form');
 const collection = 'https://www.themoviedb.org/collection/';
+const hightScore = 70;
```

Comment:
```suggestion
const highScore = 70;
```
Usa este [Plugin](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) para evitar typos.